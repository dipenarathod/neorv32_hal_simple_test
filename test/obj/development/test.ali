V "GNAT Lib v14"
M P W=8
A -Og
A -gnatA
A --RTS=/home/dipen/.local/share/alire/builds/bare_runtime_14.0.0_095db6f0/282b01b920f0d5bb2bac604ac6d9e811f26d175144bc99af963e0381e797ee94/
A -mabi=ilp32
A -ffunction-sections
A -fdata-sections
A -g
A -gnatwa
A -gnatw.X
A -gnatVa
A -gnaty3
A -gnatya
A -gnatyA
A -gnatyB
A -gnatyb
A -gnatyc
A -gnaty-
A -gnatyd
A -gnatye
A -gnatyf
A -gnatyh
A -gnatyi
A -gnatyI
A -gnatyk
A -gnatyl
A -gnatym
A -gnatyn
A -gnatyO
A -gnatyp
A -gnatyr
A -gnatyS
A -gnatyt
A -gnatyu
A -gnatyx
A -gnatW8
A -misa-spec=20191213
A -march=rv32i_zicsr_zifencei
P ZX

RN
RV NO_IMPLEMENTATION_PRAGMAS

U test%b		test.adb		b82cd5fd NE OO SU
W neorv32%s		neorv32.ads		neorv32.ali
W neorv32.uart0%s	neorv32-uart0.ads	neorv32-uart0.ali
W runtime_support%s	runtime_support.adb	runtime_support.ali
W uart0%s		uart0.adb		uart0.ali

D interfac.ads		20250921045803 9111f9c1 interfaces%s
D interrupts.ads	20250921075146 56bb57a4 interrupts%s
D neorv32.ads		20250921075146 d5039e82 neorv32%s
D neorv32-uart0.ads	20250921075146 7c81d2e1 neorv32.uart0%s
D runtime_support.ads	20250922081033 7bf0b3fb runtime_support%s
D system.ads		20250921045803 28e4acdd system%s
D s-stoele.ads		20250921045803 ccded4e8 system.storage_elements%s
D test.adb		20250922075801 d09d3eaf test%b
D uart0.ads		20250921075146 bac314ca uart0%s
G a e
G c Z b b [test standard 7 11 none]
X 3 neorv32.ads
11K9*neorv32 107e12 8|3r6 4w6
X 4 neorv32-uart0.ads
10K17*UART0 132e18 8|3w14
X 5 runtime_support.ads
3K9*Runtime_Support 6e20 8|2w6
X 8 test.adb
7U11*Test 7b11 18l5 18t9
X 9 uart0.ads
1K9*Uart0 8|5w6 9|5e10

