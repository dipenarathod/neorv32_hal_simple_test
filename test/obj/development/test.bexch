[GENERATED OBJECT FILE]
b__test.o
[PROJECT FILES]
/home/dipen/neorv32_workspace/neorv32_hal/test/test.gpr
20250922074506 633d2457d094581ca9f355cde3ad5643
/home/dipen/neorv32_workspace/neorv32_hal/config/neorv32_hal_config.gpr
20250922075828 a28932783a0e5a9b4043072ff7693c55
/home/dipen/neorv32_workspace/neorv32_hal/neorv32_hal.gpr
20250921075146 2e567302ca96f4ff654d92860ae719c4
/home/dipen/.local/share/alire/builds/bare_runtime_14.0.0_095db6f0/282b01b920f0d5bb2bac604ac6d9e811f26d175144bc99af963e0381e797ee94/target_options.gpr
20250921045803 f41afb90e77c63a9385fba3b7aa239f1
/home/dipen/.local/share/alire/builds/bare_runtime_14.0.0_095db6f0/282b01b920f0d5bb2bac604ac6d9e811f26d175144bc99af963e0381e797ee94/bare_runtime.gpr
20250921045803 be29748ec64a1d4ced472bb9ced44188
/home/dipen/neorv32_workspace/neorv32_hal/test/config/test_config.gpr
20250922075828 b21edeba8f397a3897040f7f965e5051
[BOUND OBJECT FILES]
/home/dipen/neorv32_workspace/neorv32_hal/test/obj/development/runtime_support.o
/home/dipen/neorv32_workspace/neorv32_hal/test/obj/development/test.o
[GENERATED SOURCE FILES]
b__test.ads
b__test.adb
b__test.ali
[RESULTING OPTIONS]
-L/home/dipen/neorv32_workspace/neorv32_hal/test/obj/development/
-L/home/dipen/neorv32_workspace/neorv32_hal/test/obj/development/
-L/home/dipen/.local/share/alire/builds/bare_runtime_14.0.0_095db6f0/282b01b920f0d5bb2bac604ac6d9e811f26d175144bc99af963e0381e797ee94/adalib/
-L/home/dipen/neorv32_workspace/neorv32_hal/lib/
-static
/home/dipen/.local/share/alire/builds/bare_runtime_14.0.0_095db6f0/282b01b920f0d5bb2bac604ac6d9e811f26d175144bc99af963e0381e797ee94/adalib/libgnat.a
