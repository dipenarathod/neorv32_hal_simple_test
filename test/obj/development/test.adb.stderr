test.adb:1:04: (style) space required [-gnatyt]
test.adb:2:06: warning: unit "Runtime_Support" is not referenced [-gnatwu]
test.adb:3:13: warning: unit "neorv32.UART0" is not referenced [-gnatwu]
test.adb:4:06: warning: no entities of "neorv32" are referenced [-gnatwu]
test.adb:5:06: warning: unit "Uart0" is not referenced [-gnatwu]
test.adb:8:01: (style) multiple blank lines [-gnatyu]
test.adb:11:07: (style) space required [-gnatyt]
test.adb:12:07: (style) space required [-gnatyt]
test.adb:13:01: (style) trailing spaces not permitted [-gnatyb]
test.adb:14:07: (style) space required [-gnatyt]
